skimage/measure/fit.py

```diff
@@ -699,6 +699,8 @@ def ransac(data, model_class, min_samples, residual_threshold,
         If RandomState instance, random_state is the random number generator;
         If None, the random number generator is the RandomState instance used
         by `np.random`.
+    init_inliers : array-like of bool, shape (N,), optional
```

Comment:
init makes me think of the `__inti__` callable. Should this be better off as `initial_inliers`